src/ssl.c

```diff
@@ -49032,6 +49625,30 @@ int wolfSSL_BN_is_negative(const WOLFSSL_BIGNUM* bn)
     return mp_isneg((mp_int*)bn->internal);
 }
 
+WOLFSSL_API void wolfSSL_BN_zero(WOLFSSL_BIGNUM* bn)
+{
+    if (bn == NULL) {
```

Comment:
Should also check `bn->internal`. Example our mp_zero isn't always checking NULL.